tests/test_monkeypatch.py

```diff
@@ -54,6 +61,75 @@ def test_copy_extensions(self):
         # in copy_extensions, now we call the original manager instead
         # https://github.com/divio/djangocms-versioning/pull/201/files#diff-fc33dd7b5aa9b1645545cf48dfc9b4ecR19
 
+    def test_pagecontent_copy_method_creates_extension_title_extension_attached(self):
+        """
+        The page content copy method should create a new title extension, if one is attached to it.
+        """
+        poll_extension = PollTitleExtensionFactory(extended_object=self.pagecontent)
+        poll_extension.votes = 5
+
+        new_pagecontent = copy_page_content(self.pagecontent)
+
+        self.assertNotEqual(new_pagecontent.polltitleextension, poll_extension)
+        self.assertEqual(new_pagecontent.polltitleextension.votes, 5)
```

Comment:
We should also check the pk's here as the copy method should show that the original PageContent has the Extension with Pk X and the new copied PageContent has an extension with a Pk that's not the same as before and is a new one.

This shows that a new model was created and enforces that they are different i.e. it's equal to what we expect and enforces not equal to the original.
self.pagecontent.polltitleextension.pk is equal to poll_extension.pk
new_pagecontent.polltitleextension.pk not equal poll_extension.pk